mmdet3d/models/necks/pointnet2_fp_neck.py

```diff
@@ -0,0 +1,66 @@
+from mmcv.runner import BaseModule
+from torch import nn as nn
+
+from mmdet3d.ops import PointFPModule
+from mmdet.models import NECKS
+
+
+@NECKS.register_module()
+class PointNetFPNeck(BaseModule):
+    """PointNet FP Module used in PointRCNN.
+
+    Refer to the `official code <https://github.com/charlesq34/pointnet2>`_.
+
+    Args:
+        fp_channels (tuple[tuple[int]]): Tuple of mlp channels in FP modules.
```

Comment:
Need `optional` because the arg has the default value?

Reply:
I this it is not a optional arg, because every different model will have the different setting. The default value is just a example.

Reply:
We just regard an argument optional if a default value is given. If the input value should be carefully set, we should remove the default value, or the error message will be not much straightforward.

Reply:
OK, I got it, I will delete the default value.